doc/source/whatsnew/v1.0.0.rst

```diff
@@ -679,6 +679,8 @@ Datetimelike
 - Bug in :meth:`Series.var` failing to raise ``TypeError`` when called with ``timedelta64[ns]`` dtype (:issue:`28289`)
 - Bug in :meth:`DatetimeIndex.strftime` and :meth:`Series.dt.strftime` where ``NaT`` was converted to the string ``'NaT'`` instead of ``np.nan`` (:issue:`29578`)
 - Bug in :attr:`Timestamp.resolution` being a property instead of a class attribute (:issue:`29910`)
+- Bug in :func:`pandas.core.tools.datetimes.should_cache` raising ``TypeError`` when trying to get a slice of a deque (:issue:`29403`)
```

Comment:
Could you reword to make more user facing? i.e. `to_datetime` failing for `deque`s when using the cache

Reply:
Thanks for the hint. Reworded.